app/api_topologies.go

```diff
@@ -90,81 +96,98 @@ func AddInitialTopologiesToRegistry(registry *Registry) {
 	// be the verb to get to that state
 	registry.Add(
 		APITopologyDesc{
-			id:          processesTopologyDescID,
+			id:          processesID,
```

Comment:
I think its best not to mix lots of renamings into a PR that adds features.  Makes it hard to see the real changes.